daprdocs/content/en/operations/hosting/self-hosted/self-hosted-with-docker.md

```diff
@@ -149,6 +149,10 @@ To further learn how to run Dapr with Docker Compose, see the [Docker-Compose Sa
 If your deployment target is Kubernetes please use Dapr's first-class integration. Refer to the
 [Dapr on Kubernetes docs]({{< ref "kubernetes-overview.md" >}}).
 
+## Name resolution
+
+Dapr by default uses mDNS as the name resolution component in self-hosted mode for service invocation. If running on multiple machines and/or VMs, visit the [supported name resolution providers page]({{< ref supported-name-resolution. >}}) to pick an alternate provider.
```

Comment:
Same comment as above. Something like this... "If running on multiple machines and/or VMs, use the HashiCorp Consul [supported name resolution providers page]({{< ref supported-name-resolution. >}}) name resolution component."